hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonOutputCommitter.java

```diff
@@ -302,6 +317,54 @@ private void commitJobForPartition(JobContext context, boolean overwriteSet,
     commitJobFinal(context, loadModel, operationContext, carbonTable, uniqueId);
   }
 
+  @SuppressWarnings("unchecked")
+  private void writeSegmentWithoutMergeIndex(JobContext context, CarbonLoadModel loadModel,
```

Comment:
Please add method description, as why this is required

Reply:
Added header for this private method